### ISSUE TITLE
Submit an abstract for Jena Online Symposium for Digital Education in STEM by 6 November 2022

### ISSUE THREAD

**Issue by Daniel-Mietchen** [Daniel-Mietchen/ideas#1730]
as per
https://www.asp.uni-jena.de/josdes2022
and
https://www.asp.uni-jena.de/josdes22-submission .
![Screenshot 2022-11-06 at 13-39-45 Abstract Submission form for JOSDES 2022](https://user-images.githubusercontent.com/465923/200171408-304acbf6-cb4e-46dc-92bf-fd916a688b79.png)

**Comment by Daniel-Mietchen** [comment]
Here's what I just submitted:
- Ttitle: Wikimedia projects as digital education environments
- Abstract
  - The ecosystem of websites around Wikipedia facilitates a multitude of educational activities, some of which this talk will sketch out. First, the Wikimedia projects contain collaboratively curated information about a large number of topics (including but not limited to digital and STEM education) and in multiple languages, thereby facilitating introductory education. Second, they provide pointers to additional information available elsewhere, thereby facilitating knowledge discovery and deeper engagement. Third, their content and the underlying software are entirely openly licensed, allowing for independent reuse and adaptation in lots of different ways, both within and outside the Wikimedia ecosystem. Fourth, Wikimedia projects are continuously being updated, in part by reusing or adapting openly licensed materials from other sources, in part by creating new materials, including infographics or data visualizations, textbooks, quizzes or games. Fifth, Wikimedia projects are open for participation, which provides a range of avenues for integration with educational activities. Sixth, Wikimedia projects are sustained by global communities of content contributors, software developers, community organizers, donors and others, which provides multiple paths for prospective contributors to get engaged. Seventh, they are web native and accessible via mobile and desktop devices as well as offline, so they can already be reached from many educational settings, and incorporation into additional ones (e.g. augmented or virtual reality) is not hampered by legal barriers or technological silos. Eighth, activities around this digital ecosystem can be meaningfully combined with other activities like citizen science projects or photography, both on- and offline. Ninth, a good number of examples of activities involving Wikimedia projects and primary, secondary or tertiary education have been publicly documented, and various groups within the ecosystem focus on its educational aspects. Tenth, Wikimedia projects are increasingly integrated with professional workflows in areas ranging from cultural heritage to journalism, software development, knowledge graphs, disaster management and pedagogy, thereby facilitating educational endeavours across various levels of expertise, especially in open science contexts. Besides highlighting examples for the above, the talk will also outline potential avenues for interactions between the Wikimedia ecosystem and other activities in digital and STEM education spaces.
- Keywords: collaboration, open education, cross-disciplinary education, multilingual education, Wikipedia, Wikidata, Wikimedia Commons

I got errors during the submission, so here are the corresponding screenshots:
- [Initial submission attempt](https://user-images.githubusercontent.com/465923/200179354-b67b3d0c-63da-4859-8cbe-e7a961d3d0d2.png)
- [Error message upon initial submission attempt](https://user-images.githubusercontent.com/465923/200179355-bae6ee25-1746-4f83-bfac-aa8916b91bd2.png)
- [Second submission attempt](https://user-images.githubusercontent.com/465923/200179357-68a92c2a-2ad6-4f01-8f04-89711d245f44.png)
- [Error screen after second submission attempt](https://user-images.githubusercontent.com/465923/200179358-39bf2f6a-b40a-4f6b-89de-cc6857659104.png)
